app/src/main/java/com/infomaniak/drive/ui/publicShare/PublicShareListFragment.kt

```diff
@@ -209,6 +212,29 @@ class PublicShareListFragment : FileListFragment() {
         )
     }
 
+    private fun openBookmark(file: File) {
+        publicShareViewModel.executeDownloadAction(
+            activityContext = requireActivity(),
+            downloadAction = DownloadAction.OPEN_BOOKMARK,
+            file = file,
+            navigateToDownloadDialog = {
+                withContext(Dispatchers.Main) {
+                    safeNavigate(
+                        resId = R.id.previewDownloadProgressDialog,
+                        args = PreviewDownloadProgressDialogArgs(file.name).toBundle(),
+                    )
+                }
+            },
+            onDownloadSuccess = {},
```

Comment:
Do we want to have a default value for `onDownloadSuccess` in `executeDownloadAction` ?